src/engines/quickjs.js

```diff
@@ -29,6 +31,11 @@ class QuickJSInstaller extends Installer {
   }
 
   static resolveVersion(version) {
+    if (['darwin-arm64', 'darwin-x64', 'linux-arm64'].includes(platform)) {
+      return fetch('https://api.github.com/repos/napi-bindings/quickjs-build/releases')
+        .then((r) => r.json())
+        .then((b) => b[0].tag_name);
```

Comment:
i don't think this is inherently sorted the way you might expect. see graal for example.